packages/ui/src/proposals/components/StatisticsPreview/ProposalStatistics.tsx

```diff
@@ -31,9 +31,10 @@ export const ProposalStatistics = ({ voteCount, constants }: ProposalStatisticsP
   const approvalQuorum = councilSize && approvalQuorumRatio && Math.ceil(councilSize * approvalQuorumRatio)
   const slashingQuorum = councilSize && slashingQuorumRatio && Math.ceil(councilSize * slashingQuorumRatio)
 
-  const quorumRatio = councilSize ? total / councilSize : 0
-  const approvalRatio = total && approve / total
-  const slashingRatio = total && slash / total
+  const approvalCurrentRatio = councilSize ? (total - abstain) / councilSize : 0
+  const slashingCurrentRatio = councilSize ? (total - abstain) / councilSize : 0
+  const approvalRatio = total && total - abstain && approve / (total - abstain)
+  const slashingRatio = total && total - abstain && slash / (total - abstain)
```

Comment:
Nice catch :+1:
Just one small detail (to help readability): because total is always defined this should be enough:
```suggestion
  const approvalRatio = total - abstain && approve / (total - abstain)
  const slashingRatio = total - abstain && slash / (total - abstain)
```